### PR TITLE
add refreshAll method

### PR DESCRIPTION
**Pull request by matej21**

### CODE REVIEW

File: src/Entity/AbstractEntity.php [hunk 1]
```diff
@@ -267,9 +270,16 @@ protected function onLoad(array $data)
 	protected function onRefresh(array $data)
 	{
 		foreach ($this->metadata->getProperties() as $name => $metadataProperty) {
+			if (isset($this->data[$name]) && $this->data[$name] instanceof HasMany) {
+				$this->data[$name]->clean();
+			}
 			if (!$metadataProperty->isVirtual && isset($data[$name])) {
+				if (isset($this->data[$name]) && $this->data[$name] instanceof OneHasOne) {
+					$this->data[$name]->set(NULL, TRUE);
```
Comment (hrach) [inline src/Entity/AbstractEntity.php]: lowercased null, true :)

File: src/Relationships/HasMany.php [hunk 2]
```diff
@@ -234,6 +234,15 @@ public function trackEntity(IEntity $entity)
 	}
 
 
+	public function clean()
```
Comment (hrach) [inline src/Relationships/HasMany.php]: Please put @internal phpdoc.

**Comment by hrach** [comment]
Some doc would be also nice :-)

**Comment by hrach** [comment]
Could you please fix the tests?

**Comment by matej21** [comment]
@hrach fixed

**Comment by hrach** [comment]
Merged in #227

**Pull request closed without merging**